cmd/kubeadm/app/util/runtime/runtime.go

```diff
@@ -93,6 +94,22 @@ func (runtime *CRIRuntime) Connect() error {
 	return nil
 }
 
+// Close closes the connections to the runtime and image services.
+func (runtime *CRIRuntime) Close() error {
+	var errs []error
+	if runtime.runtimeService != nil {
+		if err := runtime.runtimeService.Close(); err != nil {
+			errs = append(errs, errors.Wrap(err, "failed to close runtime service"))
+		}
+	}
+	if runtime.imageService != nil {
+		if err := runtime.imageService.Close(); err != nil {
+			errs = append(errs, errors.Wrap(err, "failed to close image service"))
+		}
+	}
+	return errorsutil.NewAggregate(errs)
```

Comment:
```suggestion
	klog.Warningf("Failed to close container runtime: %v", errorsutil.NewAggregate(errs))
```

i think this is simpler, we always print a warning anyhow.

we lose context prefixes like `[config/images]` but that's not so important.